services/ui-src/src/styles/index.scss

```diff
@@ -55,6 +55,12 @@ a {
   background-color: var(--chakra-colors-palette-gray_lightest);
 }
 
+// ChoiceList Override
+
+.ds-c-choice[type="checkbox"]:checked::after {
+  box-sizing: content-box;
+}
+
```

Comment:
We can prob do this in the `sx` in ChoiceList. I'll help.

Reply:
Yeah... this was a weird one. I decided on this method since it'll occur every single time we want to use a checkbox choicelist or choice and it felt weird to include that as an sxOveride prop every time. Or if we didn't want to do that, create another box to simply apply this sx onto. Happy to go either way tbh

Reply:
What I would do is build it out as an `sx` object in the component file, like: 

```
".ds-c-choice[type='checkbox']:checked::after":  {
  box-sizing: content-box;
}
```

and then have it style the component alongside the override, like:

```
<Component sx={{ ...sx, ...sxOverride }} />
```

Reply:
Fixed!